akka-projection-kafka/src/it/scala/akka/projection/kafka/KafkaSourceProviderSpec.scala

```diff
@@ -60,6 +61,7 @@ class KafkaSourceProviderSpec extends KafkaSpecBase() {
       first.offset() shouldBe 0L
 
       probe.cancel()
+      //provider.asInstanceOf[KafkaSourceProviderImpl[_, _]].control.foreach(c => Await.ready(c.shutdown(), 10.seconds))
```

Comment:
is this a leftover ?

Reply:
Yes, I noticed the tests weren't passing anymore, but the cause has been obfuscated in all the failures over the past few weeks.  I'm still experimenting.  In this case the test was failing because stream stages were still active after the test completed (as detected by `assertAllStagesStopped`).  The `withStopTimeout` config resolved this, but I'm not sure why it's now necessary.